test/fluree/db/query/sparql_test.cljc

```diff
@@ -254,7 +268,7 @@
                  WHERE {?e person:favNums ?favNums.}
                  GROUP BY ?e HAVING(SUM(?favNums) > 1000)"
           {:keys [groupBy having]} (sparql/->fql query)]
-      (is (= "?e"
+      (is (= ["?e"]
```

Comment:
Needs to be reverted too?

Reply:
Same deal here, the translator will always output a vector for ORDER BY, even though FQL accepts it both ways.